README.md

```diff
@@ -202,7 +202,7 @@ await PostRepository.find(
 
 ## Type mappings
 
-See [LoopBack types](http://loopback.io/doc/en/lb3/LoopBack-types.html) for details on LoopBack's data types.
+See [LoopBack types in LoopBack 4](http://loopback.io/doc/en/lb4/LoopBack-types.html) (or [LoopBack types in LoopBack 3](http://loopback.io/doc/en/lb3/LoopBack-types.html)) for details on LoopBack's data types.
```

Comment:
maybe we want to keep both at least for now?

Reply:
Yes! We list LB4 and LB3 links here.